hw2.py

```diff
@@ -0,0 +1,43 @@
+from ctypes import sizeof
+from inspect import indentsize
+from operator import index
+from unicodedata import numeric
+
+
+list = [2, 0 ,1 , 0]
+
+#Q1
+print(len(list))
+
+#Q2
+print(list[0]) #print 2
+
+#Q3
+print(list.count(0)) #print 2
+
+#Q4
+#print(list[4]) #list index out of range
+
+#Q5
+print(2 in list) #True
+
+#Q6

```

Comment:
you need just to append 'A', no need for pop()

Reply:
i need pop it because i need use same list another code without 'A'